src/use-cases/queries/RecupererMesMembres.test.ts

```diff
@@ -0,0 +1,175 @@
+import { MesMembresReadModel, MesMembresReadModelLoader, RecupererMesMembres } from './RecupererMesMembres'
+
+describe('recupererMesMembres', () => {
+  afterEach(() => {
+    mesMembresLoaderPrisma = original
+  })
+  describe('quand on demande les membres rattachés à une gouvernance existante d’un département existant', () => {
+    it('et que le membre est une préfecture alors on les récupères mais il n’est pas autorisé à être supprimer', async () => {
+      mesMembresLoaderPrisma = {
+        ...mesMembresLoaderPrisma,
+        membres: [
+          {
+            contactReferent: {
+              nom: 'Henrich',
+              prenom: 'Laetitia',
+            },
+            nom: 'Préfecture du Rhône',
+            roles: ['Co-porteur'],
+            statut: 'Membre',
+            typologieMembre: 'Préfecture départementale',
+          },
+        ],
+      }
+      const queryHandler = new RecupererMesMembres(new MesMembresReadModelLoaderStub())
+      const mesMembres = await queryHandler.get({ codeDepartementGouvernance: '69' })
+      expect(mesMembres.membres).toStrictEqual(
+        [
+          {
+            contactReferent: {
+              nom: 'Henrich',
+              prenom: 'Laetitia',
+            },
+            nom: 'Préfecture du Rhône',
+            roles: ['Co-porteur'],
+            statut: 'Membre',
+            suppressionDuMembreNonAutorise: true,
+            typologieMembre: 'Préfecture départementale',
+          },
+        ]
+      )
+    })
+
+    it('et que le membre n’est pas la préfecture alors on le récupère mais il est autoriser à être supprimer', async () => {
+      mesMembresLoaderPrisma = {
+        ...mesMembresLoaderPrisma,
+        membres: [
+          {
+            contactReferent: {
+              nom: 'Didier',
+              prenom: 'Durant',
+            },
+            nom: 'Département du Rhône',
+            roles: ['Co-porteur', 'Co-financeur'],
+            statut: 'Membre',
+            typologieMembre: 'Collectivité, EPCI',
+          },
+        ],
+      }
+      const queryHandler = new RecupererMesMembres(new MesMembresReadModelLoaderStub())
+      const mesMembres = await queryHandler.get({ codeDepartementGouvernance: '69' })
+      expect(mesMembres.membres).toStrictEqual(
+        [
+          {
+            contactReferent: {
+              nom: 'Didier',
+              prenom: 'Durant',
+            },
+            nom: 'Département du Rhône',
+            roles: ['Co-porteur', 'Co-financeur'],
+            statut: 'Membre',
+            typologieMembre: 'Collectivité, EPCI',
+          },
+        ]
+      )
+    })
+
+    it('et que le gestionnaire département n’est pas un co-porteur alors on le récupère mais je ne peux ni supprimer et ni ajouter un membre', () => {
+      ['Co-Financeur', 'Bénéficiaire', 'Formation', 'Observateur'].forEach(async (role) => {
```

Comment:
Interdit de faire des `forEach` comme ça car on ne les voit pas dans la sortie.
Tu peux utiliser `it.each`, on en a fait pas mal dans le code.